INTERNSHIP/env/Lib/site-packages/pip/_internal/commands/search.py

```
@@ -0,0 +1,174 @@
+            "--index",
+            dest="index",
+            metavar="URL",
+            default=PyPI.pypi_url,
+            help="Base URL of Python Package Index (default %default)",
+        )
+
+        self.parser.insert_option_group(0, self.cmd_opts)
+
+    def run(self, options: Values, args: List[str]) -> int:
+        if not args:
+            raise CommandError("Missing required argument (search query).")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)